### PR TITLE
Add @open-telemetry/profiling-maintainers as CODEOWNERS

### PR DESCRIPTION
**Pull request by tigrannajaryan**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (AlexanderWert) [inline .github/CODEOWNERS:28]: Seems that @open-telemetry/profiling-maintainers do not have write access to this repository?

**Review by felixge — APPROVED** [review]

File: .github/CODEOWNERS
Comment (tigrannajaryan) [reply .github/CODEOWNERS:28]: I added the permission, should be good now.

**Comment by tigrannajaryan** [comment]
@open-telemetry/technical-committee please approve the changes.

**Comment by tigrannajaryan** [comment]
Hmm, for some reasons I am unable to merge this although it shows we have 2 approvals (the required number). Weird. Can I get more approvals to see if it makes a difference?

**Review by arminru — APPROVED** [review]

**Comment by arminru** [comment]
@tigrannajaryan It was just lacking an owners approval for the CODEOWNERS file itself as per:
```
*   @open-telemetry/specs-semconv-approvers 
```

**Pull request merged**
